.github/workflows/ci.yml

```diff
@@ -28,7 +28,6 @@ jobs:
         uses: ./
         with:
           languages: java
-          service: my-service
```

Comment:
Should we also remove it from `README.md` then?

Reply:
yep!